### PR TITLE
Move configs from gitlab to github.

### PR DESCRIPTION
**Pull request by dvalinrh**
Move the test configs from gitlab to github.

### CODE REVIEW

File: config/tags.conf [hunk 1]
```diff
@@ -1,4 +1,7 @@
...
-  manager: Managers name
-  project: Project name
-  env: Environment name
+  User: Your name
+  Owner: Your name
+  Manager: Rick_Sussman
+  Project: Rhel_regression
+  Environment: Test
+  Jirald: Your name
+  TicketID: 366
```
Comment (kdvalin) [inline config/tags.conf]: Remove Rick's name, and ticket ID.

File: config/verify.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+repository_type: git_tag
+location:  https://gitlab.cee.redhat.com/dvalin
```
Comment (kdvalin) [inline config/verify.yml]: Remove internal URL

**Review by kdvalin — CHANGES_REQUESTED** [review]

**Review by kdvalin — APPROVED** [review]
LGTM

File: config/tags.conf [hunk 3]
```diff
@@ -1,4 +1,7 @@
-  full_name: Your name
-  manager: Managers name
-  project: Project name
-  env: Environment name
+  User: Your name
+  Owner: Your name
+  Manager: Managers Name
+  Project: Rhel_regression
```
Comment (frival) [inline config/tags.conf:4]: Probably should change this one too while we're at it.

**Pull request merged**